specs/config/config-resources.html

```diff
@@ -76,13 +76,16 @@
 
         // Other parts of multi-part spec
         additionalArtifacts: [
-          { title: "OSLC Configuration Management 1.0 Part 1: Overview", href: "oslc-config-mgt.html" },
-          { title: "OSLC Configuration Management 1.0 Part 2: Versioned Resources", href: "versioned-resources.html" },
+          { title: "OSLC Configuration Management 1.0: Overview", href: thisBase + "oslc-config-mgt.html" },
           {
-            title: "OSLC Configuration Management 1.0 Part 3: Configuration Specification (this document)",
-            href: "config-resources.html",
+            title: "OSLC Configuration Management 1.0: Versioned Resources",
+            href: thisBase + "versioned-resources.html",
           },
-          { title: "OSLC Configuration Management 1.0 Part 4: RDF Vocabulary", href: "config-vocab.html" },
+          {
+            title: "OSLC Configuration Management 1.0: Configuration Specification (this document)",
+            href: thisBase + "config-resources.html",
+          },
+          { title: "OSLC Configuration Management 1.0: RDF Vocabulary", href: thisBase + "config-vocab.html" },
```

Comment:
Machine readable resources should be referenced too, without part numbers. Any reason part numbers were removed from the main parts of the spec?

Reply:
I will get that on the next PSD. No need to hold up initial publishing for that.

Reply:
I added them

Reply:
Also there was a missing slash in the generated path.